### PR TITLE
APEXCORE-585 #resolve do not try to calculate latency if the first wi…

### PR DESCRIPTION
**Pull request by davidyan74**
…ndow has not complete

@vrozov please review and merge

### CODE REVIEW

File: engine/src/test/java/com/datatorrent/stram/StreamingContainerManagerTest.java [hunk 1]
```diff
@@ -1083,7 +1066,7 @@ public void testLatency() throws Exception
     GenericTestOperator o2 = dag.addOperator("o2", GenericTestOperator.class);
     HighLatencyTestOperator o3 = dag.addOperator("o3", HighLatencyTestOperator.class);
     GenericTestOperator o4 = dag.addOperator("o4", GenericTestOperator.class);
-    long latency = 5000; // 5 seconds
+    long latency = 2000; // 2 seconds
```
Comment (vrozov) [inline engine/src/test/java/com/datatorrent/stram/StreamingContainerManagerTest.java]: What is the reason to go from 5 sec to 2 sec? Was it there to account for the first window?

Comment (vrozov) [reply engine/src/test/java/com/datatorrent/stram/StreamingContainerManagerTest.java]: The latency seems to be more like a constant and not a variable. Should it be declared as private static final long?

File: engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java [hunk 2]
```diff
@@ -1838,7 +1838,7 @@ long getEndWindowEmitTimestamp(long windowId, PTOperator oper)
 
   public long updateOperatorLatency(PTOperator oper, UpdateOperatorLatencyContext ctx)
   {
-    if (!oper.getInputs().isEmpty()) {
+    if (!oper.getInputs().isEmpty() && oper.stats.currentWindowId.get() > 0) {
```
Comment (vrozov) [inline engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java:1841]: Please add unit test. StreamingContainerManagerTest.testLatency() fails intermittently, so it does not cover the change.

**Comment by davidyan74** [comment]
@vrozov thanks for the review. There was actually a bug in the original code that was uncovered by this intermittent unit test failure. it's fixed.

**Pull request merged**